src/ucp/tag/tag_match.h

```diff
@@ -39,10 +39,14 @@ typedef struct ucp_tag_match {
     } unexpected;
 
     /* Tag offload fields */
-    ucs_queue_head_t          offload_ifaces; /* Interfaces which support tag offload */
-    ucs_queue_head_t          sync_reqs;      /* Outgoing sync send requests */
-    size_t                    post_thresh;    /* Messages are posted to the transport
-                                                 starting from this threshold. */
+    ucs_queue_head_t          offload_ifaces;   /* Interfaces which support tag offload */
+    ucs_queue_head_t          sync_reqs;        /* Outgoing sync send requests */
+    size_t                    post_thresh;      /* Messages are posted to the transport
```

Comment:
s/post_thresh/thresh - "posting" is not UCT API terminology (it's IB specific) and we should avoid using it in UCP code/comments. Need to use "tag_recv", "zcopy", etc.